### PR TITLE
remove deprecated usage of getExtension

### PR DESCRIPTION
**Pull request by kesslerdev**

### CODE REVIEW

**Comment by kesslerdev** [comment]
only for twig < 1.26

File: Twig/Node/TimelineActionThemeNode.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ public function compile(\Twig_Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('echo $this->env->getExtension(\'timeline_render\')->setTheme(')
+            ->write('echo $this->env->getExtension(\'Spy\TimelineBundle\Twig\Extension\TimelineExtension\')->setTheme(')
             ->subcompile($this->getNode('action'))
```
Comment (core23) [inline Twig/Node/TimelineActionThemeNode.php]: Please don't use fcqn here

Comment (shouze) [reply Twig/Node/TimelineActionThemeNode.php]: So what do you suggest? using `timeline_render` generates a deprecation.

Comment (stephpy) [reply Twig/Node/TimelineActionThemeNode.php]: You can use TimelineExtension::class, then `use`

**Comment by stephpy** [comment]
🍻

**Pull request merged**